draft-ietf-quic-transport.md

```diff
@@ -2357,11 +2361,11 @@ shown in the following figure and described below.
                             +--------+
                                  |
                         send/recv STREAM/RST
-                             recv MSD/SB
+                             recv MSD/SB             
                                  |
                                  v
-                 recv FIN/  +--------+    send FIN/
-                 recv RST   |        |    send RST
+              recv FIN/     +--------+     send FIN/
+         recv RST/send UNI  |        |  send RST/recv UNI
```

Comment:
after send UNI should really be a new state completely, the problem is that in a normal rcv FIN/ rcv RST state it is absolutely expected to get stream frame retransmissions, however on a unidirectional stream it is illegal and should be a transport error.